backend/experiment/migrations/0054_migrate_block_content.py

```diff
@@ -0,0 +1,89 @@
+# Generated by Django 4.2.14 on 2024-08-07 14:30
+
+from django.db import migrations
+
+
+def migrate_block_content(apps, schema_editor):
+    Block = apps.get_model("experiment", "Block")
+    BlockTranslatedContent = apps.get_model("experiment", "BlockTranslatedContent")
+    Phase = apps.get_model("experiment", "Phase")
+    Experiment = apps.get_model("experiment", "Experiment")
+    ExperimentTranslatedContent = apps.get_model("experiment", "ExperimentTranslatedContent")
+
+    for block in Block.objects.all():
+        language = block.language if hasattr(block, "language") and block.language else "en"
+
+        BlockTranslatedContent.objects.create(
+            block=block,
+            language=language,
+            name=block.name,
+            description=block.description,
+        )
+
+        if block.phase:
```

Comment:
This part of the migration is also in #1240 - that also includes a reverse migration to remove the `Experiment` and `Phase` objects that would be created here. The condition for removing them, i.e., checking for a match to the auto-generated name of the phase, may cause errors though, but when I tested the migration locally, it did prove handy to roll this back. So *either* we merge #1240 first and remove this part here, or include a backwards migration for the created `Experiment`, `ExperimentTranslatedContent`, and `Phase` objects, and close #1240. (The latter is probably easier.)